payments/README.md

```diff
@@ -0,0 +1,13 @@
+# Payment Microservice
+
+A Node.js Express Application with MongoDB database.
+
+MongoDB Admin Panel can be accessed through localhost:8888, and the Express App is running on port 5008. 
```

Comment:
seems to be outdated

might also want to add [mongo compass](https://www.mongodb.com/products/compass) as an explorer for the mongodb